package.json

```diff
@@ -5,6 +5,7 @@
     "packages/*"
   ],
   "scripts": {
+    "pp": "yarn lerna exec --stream --scope @redwoodjs/cli -- jest /Users/demarco/dev/GitHub/redwood/packages/cli/src/lib/__tests__/merge.test.js --colors --maxWorkers=4",
```

Comment:
Making a note for this to be removed before PR is finalized 😉

Reply:
Ha, I almost made it to the finish line without letting this slip through. Thanks for noticing